lib/cli/collect-files.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const path = require('path');
+const ansi = require('ansi-colors');
+const debug = require('debug')('mocha:cli:run:helpers');
+const minimatch = require('minimatch');
+const utils = require('../utils');
+
```

Comment:
Could you add this "@module" and "@private" stuff, please. To be honest I don't know the purpose of it, but let's follow the line of the other modules.